packages/testing/src/execution_testing/cli/tox_helpers.py

```diff
@@ -55,6 +55,17 @@ def write_github_summary(
             f.write("```\n")
 
 
+def find_project_root() -> Path:
+    """Locate the root directory of this project."""
+    # Search upwards from file location
+    script_dir = Path(__file__).resolve().parent
+    for parent in [script_dir, *script_dir.parents]:
+        if (parent / "EIP_AUTHORS_MANUAL.md").exists():
```

Comment:
We should use a more dependable file than `EIP_AUTHORS_MANUAL.md`, e.g., `pyproject.toml`.